docs/_assets/codepen.js

```diff
@@ -36,29 +36,12 @@ CodeBlockCodePen.prototype.MDLIBS = [
 CodeBlockCodePen.prototype.init = function() {
   'use strict';
 
-  for (var i = 0, len = this.htmlCodeBlocks.length; i < len; i++) {
-    var pre = this.htmlCodeBlocks[i];
+  for (var i = 0, len = this.codepenButtons.length; i < len; i++) {
```

Comment:
Why would he yell? I thought `Array.prototype` (from reading online, not testing so don't kill me) caused no needless object to get created where `[]` will create an object that essentially does nothing.  (Ok, *one* needless object not a big deal but still...)